src/main/java/com/example/demo/categories/Category.java

```diff
@@ -0,0 +1,32 @@
+package com.example.demo.categories;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "categories")
+public class Category {
+
+    @Id
+    @SequenceGenerator(
+            name = "categories_id_seq",
+            sequenceName = "categories_id_seq",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "categories_id_seq"
+    )
+    private int id;
+    private String name;
+
+    public Category(String name) {
```

Comment:
Nothing to modify here, I'm just letting you know that once we have the CategoryDto class, we will not need this constructor, because the CategoryDto class will have only the name, without the id field. You will use it in the Controller and Service as well